test/core/crypto/crypto_store/crypto_store_test.cpp

```diff
@@ -283,10 +247,6 @@ TEST_F(CryptoStoreTest, generateSr25519KeypairStoreSuccess) {
  * @then collection of all ed25519 public keys of provided type is returned
  */
 TEST_F(CryptoStoreTest, getEd25519PublicKeysSuccess) {
-  EXPECT_OUTCOME_TRUE_MSG_1(
-      crypto_store->initialize(crypto_store_test_directory),
-      "initialization failed");
-
   EXPECT_OUTCOME_TRUE(pair1, crypto_store->generateEd25519Keypair(kBabe));
   EXPECT_OUTCOME_TRUE(pair2, crypto_store->generateEd25519Keypair(kBabe));
```

Comment:
```suggestion
  EXPECT_OUTCOME_SUCCESS(pair1, crypto_store->generateEd25519Keypair(kBabe));
  EXPECT_OUTCOME_SUCCESS(pair2, crypto_store->generateEd25519Keypair(kBabe));
```

**EXPECT_OUTCOME_SUCCESS** like **EXPECT_OUTCOME_TRUE**, but mode informative and human-readable in reports. Try to do experiment for looking.

Reply:
I will keep it in mind.